core/Util/plPNG.h

```diff
@@ -30,23 +30,17 @@ class PLASMA_DLL hsPNGException : public hsException {
 
 class PLASMA_DLL plPNG {
 private:
-    png_structp fPngReader;
-    png_structp fPngWriter;
-    png_infop   fPngInfo;
-    png_infop   fEndInfo;
 
 public:
     static void DecompressPNG(hsStream* S, void* buf, size_t size);
     static void CompressPNG(hsStream* S, const void* buf, size_t size,
                             uint32_t width, uint32_t height, int pixelSize);
 
 private:
-    plPNG() : fPngReader(), fPngWriter(), fPngInfo(), fEndInfo() { }
-    ~plPNG();
+    plPNG() { }
```

Comment:
If plPNG is no longer instantiated, perhaps the constructor could be marked deleted